utils/refchecker.py

```diff
@@ -16,19 +17,22 @@
 import ssg.utils
 
 SSG_ROOT = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
+RULES_JSON = os.path.join(SSG_ROOT, "build", "rule_dirs.json")
+BUILD_CONFIG = os.path.join(SSG_ROOT, "build", "build_config.yml")
+CONTROLS_DIR = os.path.join(SSG_ROOT, 'controls')
```

Comment:
We probably should use double quotes because we use them in all other string literals in this script.